common/web/input-processor/src/text/prediction/predictionContext.ts

```diff
@@ -188,7 +188,8 @@ export default class PredictionContext extends EventEmitter<PredictionContextEve
         this.recentAccept = false;
         this.recentRevert = true;
       }
-      return null;
+
+      return Promise.resolve(null);
```

Comment:
Why this matters:

https://github.com/keymanapp/keyman/blob/e8949831da5b71cdebcc09330a57d82779890272/web/src/engine/osk/src/banner/suggestionBanner.ts#L609-L612

Making sure we always return a Promise, even if it doesn't contain anything notable, is the simplest solution.

The Promise otherwise exists to facilitate unit testing, and that's something safe to tweak in response.

Reply:
Agree this is better and more consistent.